src/agent/onefuzz-task/src/tasks/coverage/generic.rs

```diff
@@ -148,9 +148,6 @@ impl CoverageTask {
 
         context.heartbeat.alive();
 
-        info!("report initial coverage");
```

Comment:
keep this.
We want to report the initial coverage after we read the file

Reply:
Done.